python/pyspark/pandas/tests/test_series.py

```diff
@@ -387,6 +387,14 @@ def test_isin(self):
             )
             self.assert_eq(psser.isin([1, 5, 0, None]), expected)
 
+    def test_duplicated(self):
```

Comment:
Would you mind also adding tests for numeric, Timestamp types and null cases ??

Reply:
Good idea, more test cases are added.